.github/ISSUE_TEMPLATE/prisma-cli-studio-bug-report.yml

```
@@ -0,0 +1,53 @@
+name: Prisma CLI `prisma studio` report
+description: Something not working as expected when using `prisma studio` in Prisma CLI
```

```suggestion
description: Prisma Studio is not working as expected from the Prisma CLI
```
Just for consistency for how we are outlining the description on the Data Browser side

---

Great one 👍